SConstruct

```diff
@@ -40,11 +46,13 @@ Default('./build', './release')
 
 # Additional mode options
 if mode == 'cache':
-    CacheDir(cache_dir)
-
-# Print the state of the repo at end of SCons run
-finish_command = Command( 'state_of_repo.log', [], gslab_scons.misc.state_of_repo, MAXIT=10) # From http://stackoverflow.com/questions/8901296/how-do-i-run-some-code-after-every-build-in-scons
-Depends(finish_command, BUILD_TARGETS)
-env.AlwaysBuild(finish_command)
-if 'state_of_repo.log' not in BUILD_TARGETS: 
-    BUILD_TARGETS.append('state_of_repo.log')
+    (cache_dir)
```

Comment:
That one really is a mystery to me. I fixed it back to `CacheDir(cache_dir)` a few times and for some reason it kept popping up. I'll fix it on `dev_four`.